nyaa/routes.py

```diff
@@ -153,53 +154,15 @@ def home(rss):
         if flask.g.user.is_admin:  # God mode
             query_args['admin'] = True
 
-    # If searching, we get results from elastic search
-    use_elastic = app.config.get('USE_ELASTIC_SEARCH')
-    if use_elastic and term:
-        query_args['term'] = term
-
-        max_search_results = app.config.get('ES_MAX_SEARCH_RESULT')
-        if not max_search_results:
-            max_search_results = DEFAULT_MAX_SEARCH_RESULT
-
-        # Only allow up to (max_search_results / page) pages
-        max_page = min(query_args['page'], int(math.ceil(max_search_results / float(per_page))))
-
-        query_args['page'] = max_page
-
-        query_results = search(**query_args)
-
-        if rss:
-            return render_rss('/', query_results, use_elastic=True)
-        else:
-            rss_query_string = _generate_query_string(term, category, quality_filter, user_name)
-            max_results = min(max_search_results, query_results.total)
-            # change p= argument to whatever you change page_parameter to or pagination breaks
-            pagination = Pagination(p=query_args['page'], per_page=per_page,
-                                    total=max_results, bs_version=3, page_parameter='p',
-                                    display_msg=SERACH_PAGINATE_DISPLAY_MSG)
-            return flask.render_template('home.html',
-                                         use_elastic=True,
-                                         pagination=pagination,
-                                         torrent_query=query_results,
-                                         search=query_args,
-                                         rss_filter=rss_query_string)
+    query_results = search(**query_args)
```

Comment:
If you're going to just unpack the query args here, why not just use them here directly instead of building a dict for the args?
```python
results = search(
    term=term or '',
    user=user_id,
    sort=sort or 'id',
    ...
)
```